src/addedComponents/EntityAssertionsWidget/EntityAssertionsWidget.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+
+import { TimeRange } from '@grafana/data';
+import { ComponentSize } from '@grafana/ui';
+import { usePluginComponent } from '@grafana/runtime';
+
+interface EntityAssertionsWidgetProps {
+  query: {
+    entityName?: string;
+    entityType?: string;
+    start: number;
+    end: number;
+  };
+  size: ComponentSize;
+  source?: string;
+}
+
+export type EntityAssertionsWidgetExternal = (props: EntityAssertionsWidgetProps) => ReactElement | null;
+
+interface Props {
+  serviceName: string;
+  range: TimeRange;
+}
+
+export function EntityAssertionsWidget({ serviceName, range }: Props) {
+  const { isLoading, component: EntityAssertionsWidgetExternal } = usePluginComponent<EntityAssertionsWidgetProps>(
+    'grafana-asserts-app/entity-assertions-widget/v1'
+  );
+
+  if (isLoading || !EntityAssertionsWidgetExternal) {
+    return null;
+  }
+
+  return (
+    <EntityAssertionsWidgetExternal
+      size='md'
+      source='Traces Drilldown'
+      query={{
+        start: range.from.valueOf(),
+        end: range.to.valueOf(),
+        entityName: serviceName,
```

Comment:
Might be another example where we could using OTEL's `Resource` for passing service name like in https://github.com/grafana/grafana/pull/104680#discussion_r2107188381 🤔 . Just food for thought, nothing for this PR.